### ISSUE TITLE
Grep (~) doesn't work on windows

### ISSUE THREAD

**Issue by wargio** [radare/radare2#6895]
it simply doesn't work.

```
[0x80000000]> pd 10
            0x80000000      1d000a00       j 0x80000014
            0x80000004      8c80           ld.h d15, [a8]0
            0x80000006      0000           nop
            0x80000008      8c80           ld.h d15, [a8]0
            0x8000000a      0000           nop
            0x8000000c      8c80           ld.h d15, [a8]0
            0x8000000e      0000           nop
            0x80000010      8c80           ld.h d15, [a8]0
            0x80000012      0000           nop
            0x80000014      85f12000       ld.w d1, 0xf0000020
[0x80000000]> pd 10~nop
            0x80000000      1d000a00       j 0x80000014
            0x80000004      8c80           ld.h d15, [a8]0
            0x80000006      0000           nop
            0x80000008      8c80           ld.h d15, [a8]0
            0x8000000a      0000           nop
            0x8000000c      8c80           ld.h d15, [a8]0
            0x8000000e      0000           nop
            0x80000010      8c80           ld.h d15, [a8]0
            0x80000012      0000           nop
            0x80000014      85f12000       ld.w d1, 0xf0000020
```

**Comment by radare** [comment]
Wtf is this master or last release?

**Comment by wargio** [comment]
master.

**Comment by Maijin** [comment]
May explain why lot of tests fail on windows ...

**Comment by radare** [comment]
Rconspush/pop regression maybe?

**Comment by radare** [comment]
@skuater cant reproduce. which is your environment?

**Comment by skuater** [comment]
Yes the problem is present in lastest build from git. Im looking to solve

**Comment by skuater** [comment]
fixed on  pr #6976
